tools/transformer/src/transformations/oneOf.js

```diff
@@ -117,20 +131,15 @@ function transformOneOfProperties(parentObject, api) {
 }
 
 function canApplyOneOfTransformation(obj, api) {
-  if (!obj.oneOf) {
-    return false;
-  }
-
-  if (obj[extensionKey] === extensionOneOfValue) {
+  if (obj.oneOf) {
     return true;
   }
+  return false;
 
-  const children = obj.oneOf.map((childRef) =>
-    getObjectFromReference(childRef, api)
-  );
-  const isEnum = children.reduce((isEnum, child) => isEnum && child.enum, true);
-
-  return isEnum;
+  // const children = obj.oneOf.map((childRef) =>
```

Comment:
[q] do we want to keep this? if so shall we add some comment explaining?

Reply:
I duplicated class and forgot to remove it in 2 places.